force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,38 @@
+/*
+* Account trigger should do the following:
+* 1. Set the account type to prospect.
+* 2. Copy the shipping address to the billing address.
+* 3. Set the account rating to hot.
+* 4. Create a contact for each account inserted.
+*/ 
+trigger AccountTrigger on Account (before insert, after insert) {
+    if(Trigger.isBefore & Trigger.isInsert){
+        for(Account a : Trigger.new){
+            if(a.Type == null){ 
+                a.Type = 'Prospect';
+            }
+            if(a.ShippingStreet != null || a.ShippingCity != null || a.ShippingCountry != null || a.ShippingState != null || a.ShippingPostalCode != null){
```

Comment:
Try using the ShippingAddress field to check if it's null. It might not work for the test but it's good to practice with.